include/ultra64/sptask.h

```diff
@@ -80,25 +81,25 @@ typedef struct {
     /* 0x04 */ u32 flags;
 
     /* 0x08 */ u64* ucodeBoot;
-    /* 0x0C */ u32 ucodeBootSize;
+    /* 0x0C */ size_t ucodeBootSize;
```

Comment:
I am slightly concerned about these not being explicitly `u32`, since microcode running on the RSP will access these fields with `lw`, so if a 64-bit build were ever made and this field becomes 64-bit, it would break unexpectedly. I'm not sure if this is a real concern or not but I thought I'd mention it anyway.

Reply:
Good question. I'd say size_t is the correct type, especially since we typedef'd it to 32bits. I would say the hardware register size shouldn't factor in to the types we use beyond matching.